src/components/SocialLinks.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faFacebook,
+  faTwitter,
+  faInstagram,
+  faGithub,
+} from '@fortawesome/free-brands-svg-icons';
+
+export default function SocialLinks(props) {
```

Comment:
In my opinion, we should maintain the export default in the bottom for future purposes (wrapper, other functions, ..)

Reply:
Normally we do the arrow function like this: `SocialLinks = (props) => { code }`
In this case we are passing all the props down; we often avoid passing all props, for clarity mostly:
`.. SocialLinks = ({facebook, twitter, instagram, github}) => { code }`

Reply:
Should I do `const SocialLinks = ({facebook, twitter, instagram, github}) => { code }`, then? (_My question is the keyword before SocialLinks_)

Reply:
Yes, creating a _arrow function component_.  The arrow syntax is optional, some people say it's more clean, scope safety and compact. We should define our default syntax (personally I prefer the fat arrow)
But in the end, these components do the same:
`const foo = () => (...);`
`function foo() {
  return ...;
}`

Reply:
Tell me if it's better now :pray: